src/server/qgis_mapserver.cpp

```diff
@@ -243,6 +243,7 @@ class TcpServerWorker: public QObject
                 throw HttpException( QStringLiteral( "HTTP error unsupported method: %1" ).arg( methodString ) );
               }
 
+              // cppcheck-suppress containerOutOfBounds
```

Comment:
It raise a warning with index 2 or 1 but not with 0 just above (methodString) ?

It seems that this warning raise too much false positive, should we not just disable/not consider this check?

Reply:
There are some disappointing false positives here indeed. But then the test also did flag a number of very valid issues. I'd lean toward keeping for this reason... Even a 50% false positive rate is worth the early identification of these serious issues.